index.js

```diff
@@ -27,7 +33,9 @@ module.exports = {
 	// Get all values starting with a certain pathname and filter their views
   getAll: async function getAll(options) {
     const data = {}
-    const keys = (await module.exports.keys()).filter(key => key.startsWith(options.pathname))
+    const keys = (await module.exports.keys()).filter((key) => {
+      return options.wildcard ? key.match(keyRegex(options.pathname)) : key.startsWith(options.pathname)
```

Comment:
You wouldn't have known this because I haven't don't the analytics side, but I did not plan on sending the query param down. I planned on sending the bool value down to it based on the query param. As a practice, we should always send down actual bool values instead of having any type of adapter have to worry about that nuance